app/_src/guides/consumer-producer-policies.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Namespace scoped policies
+---
+
+## Prerequisites
+
+- Completed [quickstart](/docs/{{ page.version }}/quickstart/kubernetes-demo/) to set up a zone control plane with demo application
+
+## Basic setup
+
+In order to be able to fully utilize namespace scoped policies you need to use real [MeshService](/docs/{{ page.version }}/networking/meshservice). 
+To enable MeshService resource generation on Mesh level we need to run:
+
+```shell
+echo "apiVersion: kuma.io/v1alpha1
+kind: Mesh
+metadata:
+  name: default
+spec:
+  meshServices:
+    enabled: Everywhere
+  mtls:
+    enabledBackend: ca-1
+    backends:
+    - name: ca-1
+      type: builtin" | kubectl apply -f -
+```
+
+We also need HostnameGenerator for easy access to our services:
+
+```shell
+echo 'apiVersion: kuma.io/v1alpha1
+kind: HostnameGenerator
+metadata:
+  name: all
+  namespace: {{site.mesh_namespace}}
+  labels:
+    kuma.io/mesh: default
+spec:
+  selector:
+    meshService:
+      matchLabels:
+        k8s.kuma.io/namespace: kuma-demo
+  template: "{{ .DisplayName }}.{{ .Namespace }}.mesh"' | kubectl apply -f -
+```
```

Comment:
Isn't there already a hostnameGenerator now?

Reply:
we only have default hostNameGenerator for MES